### ISSUE TITLE
Invalid handle errors on loading SteamVR manually in a build

### ISSUE THREAD

**Issue by ZardenSawtooth** [ValveSoftware/steamvr_unity_plugin#374]
Unity version: 2017.4.3f1
SteamVR ver: 1.2.10
Unity Plugin ver: 2.2.0

I recently updated the steamVR plugin from v1.2.3 to v2.2.0. After going through and generating the Input bindings and accessing it through the code, everything works fine in the editor but I am getting the following error in the build every frame.

```<b>[SteamVR]</b> GetDigitalActionData error (/actions/simdefault/in/teleport): InvalidHandle handle: 1152974727841579526```

In both the editor and the build steamVR is initialized manually through the code. The actions get loaded and everything else seems to be working the same in the editor as in the build. Any help would be greatly appreciated.

**Comment by ZardenSawtooth** [comment]
When I go to Devices > Configure Controller in SteamVR and look at the app there it says my app is using legacy bindings in the build but the new bindings when run with the editor. Any idea what might be causing this? Looks like this might be related to my original problem.

**Comment by ZardenSawtooth** [comment]
Figured out that the issues were being caused by using the old way to initialize SteamVR that caused it behave like a legacy application. Was fixed when I used the code from the sample project included to force turn on SteamVR manually. Still don't know why it worked in the editor and failed in the build but it works both places now. Additionally, the problem was compounded by the [SteamVR] object not getting included in the DontDestroyOnLoad which caused SteamVR to break when I switched scenes.  In **SteamVR_Behaviour**, _line 63,_ it doesn't add [SteamVR] to the DontDestroyOnLoad group when it is initialized dynamically. Fixed it by updating the code as below:

```
63   if (steamVRObject == null)
64   {
65       GameObject objectInstance = new GameObject("[SteamVR]");
66       _instance = objectInstance.AddComponent<SteamVR_Behaviour>();
67       _instance.steamvr_render = objectInstance.AddComponent<SteamVR_Render>();
68       objectInstance.AddComponent<DontDestroyOnLoad>();
69   }
